### DISCUSSION TITLE
PCB

### DISCUSSION THREAD

**Discussion by micha-uhl** [TomNisbet/TommyPROM#74] [General]
Do you sell the PCB for the simple version?

**Comment by TomNisbet** [comment]
I don't sell any boards, but all of the files needed to make one are in the repo.  You can zip up the contents of schematics/TommyPROM/plot and upload that file to jlcpcb.com.  That's where I had mine done.  It took less than two weeks to have them manufactured and shipped to me.

**Reply by micha-uhl** [reply]
Thank you, I got the PCBs from PCBway and the promer works! But the LED does not light, also not the onboard LED "L"
With the Blink sketch it works

**Reply by TomNisbet** [reply]
The LED connected to D13 doesn't do anything with the TommyPROM code.  I added it to the PCB because I use the same hardware with a sketch that burns the microcode for my NQSAP 8-bit computer.  That sketch just burns the chip as soon as the Arduino powers up, so the LED was handy to see if the write completed successfully without having to hook up a terminal.  This was useful because I needed to burn several copies of the chip at once.

WIth the TommyPROM code, the D13 line is used to support 74LS595 shift registers in place of the 74LS164s that are used by default.  Since you aren't using those, you could modify the code to light the LED to indication a completion or failure or an other condition.
